js/messages.js

```diff
@@ -0,0 +1,51 @@
+const CSV_FILENAME = "../Test Card List CSV.csv";
+
+function getCSVData(callback = undefined) {
+    Papa.parse(CSV_FILENAME, {
+      download: true,
+      //To treat the first row as column titles
+      header: true,
+      complete: (result) => {
+        if (callback) {
+          callback(result);
+        }
+      },
+    });
+  }
+
+function renderMessages () {
+    let messageContainer = document.getElementById("artist-writer-messages");
+    for (var i = 1; i < 100; i++) {
+      let messageNode = document.createElement("li");
+      let messageArticle = document.createElement("article");
+      // TODO: Dummy data for now. Alternate roles for now as a test. This should be pulled from a csv somewhere.
+      let userRole = i % 2 === 0 ? "Artist" : "Writer";
+      let userSocial = `user_social_${i}`;
+      let h3String = [userRole, userSocial].join(" | ");
+      messageNode.tabIndex = 0;
+      messageArticle.tabIndex = 0;
```

Comment:
Remove - keyboard focus is only for interactive elements like buttons and links. A keyboard user can use arrow keys to scroll the page. Excessive tabbable elements will keep keyboard users from reaching the interactive ones

Would this happen to be interfering with the list position being read? If that's not it, my guess is the nesting of article in an li is unconventional. I would feel okay with removing the article tag nesting for just li if I felt list position was important to convey to a user. If I had more resources I would test on real users and go with the option they say prefer, but in the absence of that, using one or the other of li or article is reasonable to me.

Reply:
I **just** realized I was using voiceover wrong LOL. Was tapping the tab key and not using the VO-modifier w/ arrow keys. Sorry about that. Removed the tabIndexes. I am now hearing/seeing voiceover narrate each li with its appropriate position.